### DISCUSSION TITLE
Problems with deleting rows in Tables

### DISCUSSION THREAD

**Discussion by eyduh** [Budibase/budibase#7707] [Help] [answered]
Hi,

I'm not sure this is a bug or a misconfiguration on my part.

I have a simple table view of a form and would like to allow users to download a CSV of the entire table, as well as make some simple editing of the rows such as deleting duplicate entries. 

Exporting works fine, but if I delete a row I think it stays selected somehow and interferes with other operations:

I set up a table like so:
![image](https://user-images.githubusercontent.com/29815625/189352702-f16001f4-7b06-4132-9cb7-50c6baa8610d.png)

and the delete button like so:
![image](https://user-images.githubusercontent.com/29815625/189352971-a3e88f9a-dfac-4a6f-85fd-937ba152c427.png)

if I go into the exported app and delete a row and then try to delete another row I get `missing` in red:

https://user-images.githubusercontent.com/29815625/189354941-2e483084-f608-4921-b448-c82ec4898076.mp4

when I reload the page and try to delete the same row that gave me "missing" it works fine.

Do I need to add something to the button actions or is this a bug?

Cheers!

**Comment by melohagan** [comment, marked as answer]
Hi @eyduh 

Thanks very much for the detail! I was able to recreate this bug which I've raised here: https://github.com/Budibase/budibase/issues/7760
